build/conf/local.conf

```diff
@@ -265,7 +265,7 @@ CONF_VERSION = "1"
 # Delete the the source/object/binary files once a package is built to preserve disk space
```

Comment:
Should `rm_work` be added directly under this comment instead?

It looks like that's how things were back in https://github.com/Opentrons/oe-core/commit/eaa5c3d595a695127c2a7fb71ab4e1570a9dcad3#.

Reply:
Oh yeah we should move the include or the comment good catch

Reply:
good catch, moved it down.